docs/howto/django.rst

```diff
@@ -5,36 +5,99 @@ Install procrastinate with:
 
 .. code-block:: console
 
-    $ (venv) pip install 'procrastinate[django]'
+    $ (venv) pip install procrastinate 'procrastinate[django]'
 
-This tells pip to consider the extra dependencies from the group of dependencies named
-``django``. For now, this group only contains Django itself, which you likely already
-have in your project's dependencies. So why bother?
+This tells pip to install procrastinate but also to consider the extra
+dependencies from the group of dependencies named ``django``. For now, this
+group only contains Django itself, which you likely already have in your
+project's dependencies. So why bother?
 
-Specifying your dependency to the "``django`` extras" will ensure that your Django
-version and the one we support stay in sync through time (for now, we support every
-version, but if we learn of strong incompatibilities, we'll update the lib: we're
-considering every version is compatible until proven otherwise). Also, while this is not
-the case today, if our Django integration ever requires other third-party packages, they
-will be added here.
+Specifying your dependency to the "``django`` extras" will ensure that your
+Django version and the one we support stay in sync through time (for now, we
+support every version, but if we learn of strong incompatibilities, we'll
+update the lib: we're considering every version is compatible until proven
+otherwise). Also, while this is not the case today, if our Django integration
+ever requires other third-party packages, they will be added here.
 
-Add Procrastinate in your ``INSTALLED_APPS``::
+Add procrastinate Django app to your ``INSTALLED_APPS``:
+
+.. code-block:: python
 
     INSTALLED_APPS = [
         ...
         "procrastinate.contrib.django",
     ]
 
-Configure your procrastinate app from your Django settings::
 
-    import procrastinate
+After that, you need to create a new directory to contain the procrastinate
+app, where you will define your tasks. This app will run independently of
+Django but will be able to access the whole Django ecosystem (it's not a Django
+app, nor will be inside any of your existing Django apps). Create a new
+directory ``tasks`` and fill the ``tasks/__init__.py`` with:
+
+.. code-block:: python
+
+    import django
+    django.setup()  # Setup Django inside the worker so you can import/use ORM etc.
+
+    from procrastinate import App, AiopgConnector
```

Comment:
It's not a requirement or anything but I prefer importing whole modules rather than objects from within the modules when possible (it's not an absolute rule). It helps a lot when you need to mock, and it makes it obvious where things come from in the rest of the code. Any reason you changed that ? I'm open to changing my mind if you want to challenge this :)

Reply:
If I import the whole package, the line:
```python
    app = App(connector=AiopgConnector(**connector_params()))
```
would be:
```python
    app = procrastinate.App(connector=procrastinate.AiopgConnector(**connector_params()))
```
or even worse, considering `procrastinate.contrib.django`:
```python
    app = procrastinate.App(connector=procrastinate.AiopgConnector(**procrastinate.contrib.django.connector_params()))
```
You probably can't read it without horizontal scroll. Ok, we could split in more lines:

```python
    app = procrastinate.App(
        connector=procrastinate.AiopgConnector(**procrastinate.contrib.django.connector_params())
    )
```
But anyway it takes more time to get the whole goal of this line. Since the idea is to show the reader how to do things and make it readable (it's a code inside a documentation), I think we could use this "shortcut", even if your linting process would change it in the future. I think the way I changed the line makes more clear the concepts of App, connector and connector params.